src/test/java/org/codehaus/plexus/components/io/attributes/PlexusIoResourceAttributeUtilsTest.java

```diff
@@ -200,119 +130,18 @@ public void testSrcResource()
 
         assertTrue(pr.getOctalMode() > 0);
     }
-    public void testPermissionDenied()
+    public void testNonExistingFolder()
```

Comment:
Sorry,  I'm not quite sure I follow you. Do you mean that in Unix there are 'directories' not 'folders' and that's why the method name should be`'testNonExistingDirectory`? I choose this name because it's consistent with the rest of the existing test names (such as `testFolderJava7`).